### ISSUE TITLE
No chat id found for this token, please check the token and make sure you are pasting it in the right chat boot or group in Telegram

### ISSUE THREAD

**Issue by NageshS09** [yrestom/erpnext_telegram#44]
Hi Team,

When we are trying to get chat ID using telegram user settings we are getting such issue. Also As mentioned in the steps when we are clicking on generate telegram token no new tab is getting opened. Kindly look at the issue and guide us.

Attached Screen Recording for your reference.
[Uploading Screencast from 2024-09-26 19-38-58.webm…]()

**Comment by BleronOs** [comment]
yes also i have the same problem like you, i hope his gonna fix it

**Comment by tajff** [comment]
It's working,

ERPNext: v15.49.3 (version-15)
Erpnext Telegram Integration: v1.3.0 (version-14)
Frappe Framework: v15.53.0 (version-15)
